### PR TITLE
Create agilent34970A.py

### PR DESCRIPTION
**Pull request by cmefryday**
Initial Commit

### CODE REVIEW

File: pymeasure/instruments/agilent/agilent34970A.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+from time import time
+from pyvisa.errors import VisaIOError
+
+# Capitalize string arguments to allow for better conformity with other WFG's
+def capitalize_string(string: str, *args, **kwargs):
+    return string.upper()
+
+
+# Combine the capitalize function and validator
+string_validator = joined_validators(capitalize_string, strict_discrete_set)
```
Comment (Endever42) [inline pymeasure/instruments/agilent/agilent34970A.py]: I am not sure, this actually works the way you intend it to work... see also here: https://github.com/ralph-group/pymeasure/pull/209#discussion_r306018070

**Comment by tinix84** [comment]
should I merge my agilent34970A driver here?

**Comment by cmefryday** [comment]
Yes, merge them here.

**Comment by bmoneke** [comment]
Can we close this old and very initial PR (no properties)? @bilderbuchi

**Comment by bilderbuchi** [comment]
Yes, this has remained very incomplete. We could always pick it up again should interest arise.

**Pull request closed without merging**